influence_benchmark/vectorized_environment/environment_queue.py

```diff
@@ -0,0 +1,94 @@
+import copy
+import random
+from multiprocessing import Queue, Value
+from typing import Optional
+
+from influence_benchmark.environment.character import Character
+from influence_benchmark.environment.environment import Environment
+from influence_benchmark.environment.preference_model import PreferenceModel
+from influence_benchmark.environment.transition_model import TransitionModel
+from influence_benchmark.root import PROJECT_ROOT
+from influence_benchmark.utils.utils import load_yaml
+
+
+def get_environment_queue(env_args: dict, num_devices: int, total_env: Optional[int] = None):
+    config_path = PROJECT_ROOT / "config" / "env_configs" / env_args["env_name"]
+
+    total_environments = total_env if total_env is not None else 0
+    environment_queue = Queue()
+    if config_path.is_dir():
```

Comment:
Why does this condition being true mean that we're in multi mode?

Reply:
I guess because it's a whole directory of configs rather than a single config?

Reply:
yes exactly